var/spack/repos/builtin/packages/bedtools2/package.py

```diff
@@ -15,11 +15,8 @@ class Bedtools2(Package):
     homepage = "https://github.com/arq5x/bedtools2"
     url      = "https://github.com/arq5x/bedtools2/archive/v2.26.0.tar.gz"
 
+    version('2.29.2', sha256='bc2f36b5d4fc9890c69f607d54da873032628462e88c545dd633d2c787a544a5')
     version('2.27.1', sha256='edcac089d84e63a51f85c3c189469daa7d42180272130b046856faad3cf79112')
-    version('2.27.0', sha256='e91390b567e577d337c15ca301e264b0355441f5ab90fa4f971622e3043e0ca0')
-    version('2.26.0', sha256='15db784f60a11b104ccbc9f440282e5780e0522b8d55d359a8318a6b61897977')
-    version('2.25.0', sha256='159122afb9978015f7ec85d7b17739b01415a5738086b20a48147eeefcf08cfb')
-    version('2.23.0', sha256='9dacaa561d11ce9835d1d51e5aeb092bcbe117b7119663ec9a671abac6a36056')
```

Comment:
Is there a reason to remove the older versions?

Reply:
Minor versions are all bug fixes, as towards newer version `2.27+`, early ones become more or less obsolete.

Reply:
We usually keep the older versions anyway in case someone wants to be able to reproduce an older installation. We don't have a great system in place for deprecating software at the moment. See #15896.

Reply:
No worries. Reverted back to include all versions. Thx @adamjstewart